CSIRO/CARS/incoming_handler.sh

```diff
@@ -15,7 +15,7 @@ regex_filter() {
 # $1 - file to handle
 main() {
     local file=$1; shift
-    local regex="^CARS[[:digit:]]{4}_(World_monthly|Australia_weekly)\\.nc$"
+    local regex="^CARS[[:digit:]]{4}_(World_monthly|Australia_weekly).*\\.nc$"
```

Comment:
I actually checked on my pobox and it worked. Both original and new monthly files can be harvested (not at the same time though because there is a constraint on the unique time value per collection set up in the harvester). 

The thing is we still need to do a po_s3_del to unharvest the original file from the generic timestep schema and then remove the file from S3. Afterwards, we will need to do a async-upload (to bypass the pipeline) of the previous format so it can be back on S3. 

The po_s3_del command relies on the regexp set up in the chef(and chef-private) repo.   
it would be good to have a similar regex as in https://github.com/aodn/chef/pull/536/files. 
if we change the regepx in the chef repo(s) as suggested in the comments you've just added, we wouldn't be able to call po_s3_del.

I'm easy, just decide the order you want things to be done.